pkg/lib/serverTime.js

```diff
@@ -32,6 +32,8 @@ import { useObject, useEvent } from "hooks.js";
 
 import * as service from "service.js";
 import * as timeformat from "timeformat.js";
+import * as python from "python.js";
+import get_timesync_backend_py from "raw-loader!./get-timesync-backend.py";
```

Comment:
Does this have to be in pkg/lib? This gets shared between a lot of other projects, and also cockpit-components-shutdown. Configuring NTP (overview page only) and determining current server time (shared between lots of components/projects) should be kept separate. Or is that too hard to do?

(Fine for  a follow-up as well)

Reply:
Let's keep this as a follow up.